site2/website-next/src/theme/Navbar/index.js

```diff
@@ -307,13 +307,7 @@ function Navbar() {
               setVersion(latestStableVersion);
             }}
           />
-          <BrowserOnly>
-            {() => (
-              <a className="font-bold underline mr-4 -ml-4" href="/versions/">
-                {getVersion() == "master" ? "next" : getVersion()}
-              </a>
-            )}
-          </BrowserOnly>
+        
```

Comment:
why remove this? it's needed

Reply:
@urfreespace  On the website weekly meeting yesterday, it was decided this would be removed